src/coveralls.erl

```diff
@@ -69,38 +69,41 @@
 %%      Note that this function will crash if the modules mentioned in
 %%      any of the `Filenames` are not availabe on the node.
 %% @end
--spec convert_file(string() | [string()], string(), string()) -> string().
-convert_file(Filenames, ServiceJobId, ServiceName) ->
-  convert_file(Filenames, ServiceJobId, ServiceName, #s{}).
+-spec convert_file(string() | [string()], string(), string(), string()) ->
+                          string().
+convert_file(Filenames, ServiceJobId, ServiceName, RepoToken) ->
+  convert_file(Filenames, ServiceJobId, ServiceName, RepoToken, #s{}).
 
 %% @doc Import and convert cover files `Filenames` to a json string and send the
 %%      json to coveralls.
 %% @end
--spec convert_and_send_file(string() | [string()], string(), string()) -> ok.
-convert_and_send_file(Filenames, ServiceJobId, ServiceName) ->
-  convert_and_send_file(Filenames, ServiceJobId, ServiceName, #s{}).
+-spec convert_and_send_file(string() | [string()], string(), string(),
+                            string()) -> ok.
+convert_and_send_file(Filenames, ServiceJobId, ServiceName, RepoToken) ->
+  convert_and_send_file(Filenames, ServiceJobId, ServiceName, RepoToken, #s{}).
 
 %%=============================================================================
 %% Internal functions
 
-convert_file([L|_]=Filename, ServiceJobId, ServiceName, S) when is_integer(L) ->
-  convert_file([Filename], ServiceJobId, ServiceName, S);
-convert_file([[_|_]|_]=Filenames, ServiceJobId, ServiceName, S) ->
+convert_file([L|_]=Filename, ServiceJobId, ServiceName, RepoToken, S) when is_integer(L) ->
+  convert_file([Filename], ServiceJobId, ServiceName, RepoToken, S);
+convert_file([[_|_]|_]=Filenames, ServiceJobId, ServiceName, RepoToken, S) ->
   ok               = lists:foreach(
                        fun(Filename) -> ok = import(S, Filename) end,
                        Filenames),
   ConvertedModules = convert_modules(S),
+
   Str              =
-    "{~n"
+    "{~n" ++ RepoToken ++
```

Comment:
In the previous commit the string in the sent json was repo_token: Token. Now it seems to be only Token. Why was this changed?